charts/jaeger/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 1.17.0
+appVersion: 1.18.0
```

Comment:
Please revert this, `appVersion` refers to the version of Jaeger, what you want to bump up is `version`.

Reply:
Thanks, @naseemkullah . I will fix this

Reply:
Bumped up chart version to 0.23.0